tools/importer/src/main/java/com/vesoft/nebula/tools/reader/CsvReader.java

```diff
@@ -0,0 +1,102 @@
+package com.vesoft.nebula.tools.reader;
+
+import com.google.common.collect.Lists;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.log4j.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class CsvReader implements Reader {
+    private static final Logger LOGGER = Logger.getLogger(CsvReader.class);
+
+    private String filePath;
+    private Integer size;
+    private CSVParser csvParser;
+    private Iterator<CSVRecord> iter;
+
+    public CsvReader(String filePath, Integer size) {
+        this.filePath = filePath;
+        this.size = size;
+    }
+
+    public void init() throws Exception {
+        if (filePath.toLowerCase().startsWith("hdfs://")) {
+            dfsProcessor();
+        } else {
+            localProcessor();
+        }
+
+        this.iter = csvParser.iterator();
+    }
+
+    private void localProcessor() throws IOException {
+        Path path = Paths.get(filePath);
+        if (!Files.exists(path) || !Files.isReadable(path)) {
+            throw new IllegalArgumentException(filePath + " is not exist or not readable");
+        }
+        LOGGER.info(String.format("Reading Local FileSystem: %s", filePath));
+        this.csvParser = CSVParser.parse(
+                new File(filePath),
+                Charset.forName("UTF-8"),
+                CSVFormat.DEFAULT
+                        .withFirstRecordAsHeader()
```

Comment:
Should make user to configure whether the **first record line** is header

Reply:
The format seems different when the csv file is dump from mysql/oracle/mongo, we need to support them. We should enhance it when we implement the go-importer.